var/spack/repos/builtin/packages/seqfu/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Seqfu(Package):
+    """seqfu - Sequece Fastx Utilities"""
+
+    homepage = "https://github.com/telatin/seqfu2"
+    url = "https://github.com/telatin/seqfu2/archive/refs/tags/v1.20.3.tar.gz"
+
+    license("GPL-3.0", checked_by="dialvarezs")
+
+    version("1.20.3", sha256="1b287b99f3f1ac7045f4d551e781d6780ce168ba8e0a7bfaa0f5490f32e15938")
+
+    depends_on("nim@2", type="build")
+    depends_on("zlib", type="build")
+
+    patch("wno_incompatible_pointer_types.patch", when="%gcc@14:")
```

Comment:
Any chance of getting this upstreamed to the repository?

Reply:
I can open a pr in the repository and see